src/styles/header.scss

```diff
@@ -0,0 +1,16 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
```

Comment:
It's not the best practice to give fixed heights to blocks; it would be better to add the necessary padding.